src/data.js

```diff
@@ -26,10 +26,11 @@ var Client = Entity.configure({
     expires:        Entity.types.Date,
     /**
      * Details object with properties:
-     * - created          // Time when client was created
-     * - lastModified     // Last time client was modified
-     * - lastDateUsed     // Only updated if more than 6 hours out of date
-     * - lastRotated      // Last time accessToken was reset
+     * - created            // Time when client was created
+     * - lastModified       // Last time client was modified
+     * - lastDateUsed       // Only updated if more than 6 hours out of date
+     * - lastRotated        // Last time accessToken was reset
+     * - deleteOnExpiration // If true, can be deleted after expiration
```

Comment:
Just add:
```js
configure({
  version:          2,
  signEntities:     true,
  properties: {
    clientId:       Entity.types.String,
    description:    Entity.types.Text,
    accessToken:    Entity.types.EncryptedText,
    expires:        Entity.types.Date,
    /**
     * Details object with properties:
     * - created            // Time when client was created
     * - lastModified       // Last time client was modified
     * - lastDateUsed       // Only updated if more than 6 hours out of date
     * - lastRotated        // Last time accessToken was reset
     * - deleteOnExpiration // If true, can be deleted after expiration
     * (more properties may be added in the future)
     */
    details:        Entity.types.JSON,
    scopes:         Entity.types.JSON,  // new in v2
    disabled:       Entity.types.Number // new in v2
  },
  migrate(item) {
    item.details = _.defaults({}, item.details, {deleteOnExpiration: false});
    return item;
  }
});
```
Note: when modifying `item` I think we have to create a new value... I don't remember if azure-entities does a clone, it might :)
(Just the property, I'm pretty sure `item` may be modified)